srcpkgs/apcupsd/files/apcupsd-powerfail.sh

```diff
@@ -0,0 +1,13 @@
+PWRFAILDIR=$( awk '$1 == "PWRFAILDIR" {print $2; exit}' /etc/apcupsd/apcupsd.conf )
+PWRFAILDIR=${PWRFAILDIR:=/etc/apcupsd}
+
+if [ -f "${PWRFAILDIR}/powerfail" ]; then
+   echo
+   echo "APCUPSD will now power off the UPS"
+   echo
+   /etc/apcupsd/apccontrol killpower
+   echo
+   echo "Please ensure that the UPS has powered off before rebooting"
+   echo "Otherwise, the UPS may cut the power during the reboot!!!"
+   echo
```

Comment:
use `msg` (from `/etc/runit/functions`) and we shouldn't have all the extra blank lines

```
msg "Powering off UPS with APCUPSD..."
```
not sure the other message is necessary

Reply:
Done. I agree, the user is unlikely able to read and act upon the other message.